source/config.rst

```diff
@@ -3,48 +3,34 @@
 Configuration
 =============
 
-Most of the application is configured with the symfony parameters.yml
-file in app/config/parameters.yml.
+Most of the application is configured with the symfony .env file present in the project directory.
+1. You can find this file by listing out all hidden files in the directory using the command below.
 
-.. code-block:: yaml
-   
-   parameters:
-     # database configuration
-     database_host: 127.0.0.1
-     database_port: null
-     database_name: btd
-     database_user: btd
-     database_password: abc123
+.. code-block:: bash
+
+  ls -al
 
-     # mailer configuration
-     mailer_transport: smtp
-     mailer_host: 127.0.0.1
-     mailer_user: null
-     mailer_password: null
+2. Copy the .env file to .env.local using the below command.
 
-     # A secret key that's used to generate certain security-related tokens
-     secret: d2d31e391ebe8e7f307f64f1348d2c84b030cc66
+.. code-block:: bash
 
-     # Router and cookie information
-     router.request_context.scheme: http
-     router.request_context.host: example.com
-     router.request_context.base_url: /path/to/application
+  cp .env .env.local
 
-     # words in a generated excerpt
-     nines_blog.excerpt_length: 50
+3. Edit this .env.local file using any editor to make the below configuration changes.
 
-     # number of posts to show on the home page
-     nines_blog.homepage_posts: 3
+.. code-block:: yaml
```

Comment:
This is not yaml, and shouldn't be highlighted like yaml. You could call it bash - the syntax is the same even though it isn't a bash file.